server/recceiver/cfstore.py

```diff
@@ -493,38 +489,34 @@ def __updateCF__(processor: CFProcessor, recordInfoByName: Dict[str, RecordInfo]
     iocid = ioc_info.ioc_id
 
     if iocid not in iocs:
-        _log.warning("IOC Env Info not found: {iocid}".format(iocid=iocid))
+        _log.warning("IOC Env Info %s not found in ioc list: %s", ioc_info, iocs)
```

Comment:
Can we _please_ improve this msg/phrasing while we're at it? I _hate_ "IOC Env Info"